test/integration/RouteProxyMethodTest.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+require('../TestHelper');
+const async = require('async');
+const express = require('express');
+const MockYeahServer = require('../../server');
+const supertest = require('supertest');
+
+describe('Route proxy', () => {
+  let mockyeah;
+  let proxiedApp;
+  let proxiedServer;
+  let request;
+
+  before(done => {
+    async.parallel(
+      [
+        cb => {
+          mockyeah = MockYeahServer({}, cb);
+          request = supertest(mockyeah.server);
+        },
+        cb => {
+          proxiedApp = express();
+          proxiedApp.get('/foo', (req, res) => res.sendStatus(200));
+          proxiedServer = proxiedApp.listen(8888, cb);
```

Comment:
If you set port to `0` it will pick the first open port helping resolve possible port conflicts.

Reply:
@ryanricard Good call. I've updated the test to dynamically select the port and use it.